library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -81,6 +82,7 @@ class Client() {
     lateinit var conversations: Conversations
     var logger: XMTPLogger = XMTPLogger()
     var libXMTPClient: FfiXmtpClient? = null
+    val libXMTPVersion: String = getVersionInfo()
```

Comment:
Do you think it would be helpful to have this field on here so we can get it on Client directly? Or just the log? @richardhuaaa

Reply:
Don't have a strong opinion, that works!